includes/Admin/Admin_Page.php

```diff
@@ -30,13 +49,44 @@ public function add_hooks() {
 	 * @since n.e.x.t
 	 */
 	public function add_page() {
-		add_management_page(
+		$hook = add_management_page(
 			__( 'Plugin Check', 'plugin-check' ),
 			__( 'Plugin Check', 'plugin-check' ),
 			'activate_plugins',
 			'plugin-check',
 			array( $this, 'render_page' )
 		);
+
+		add_action( "load-{$hook}", array( $this, 'initialize_page' ) );
```

Comment:
Not strictly necessary, but a good practice would be to return the `$hook` from this method. This way you can cover the last bit here in the `test_add_page()` unit test (which is currently not tested).